CMakeLists.txt

```diff
@@ -220,6 +219,19 @@ target_compile_definitions      (libtesseract
 )
 set_target_properties           (libtesseract PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS True)
 endif()
+if (Tesseract_USE_OPENCL)
+    target_compile_definitions  (libtesseract PRIVATE USE_OPENCL=1)
+    if (APPLE OR ANDROID)
+        # Inbuilt to NDK
+        find_package(OpenCL REQUIRED)
+        target_link_libraries       (libtesseract ${OpenCL_LIBRARIES})
```

Comment:
CMake 3.0 is a general requirement for Hunter, requirement for package can be differ. We have no choice here since with `OpenCL_LIBRARIES` it will not work correctly anyway.